JSONPlaceholderAPIClient/Post.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace JSONPlaceholderAPIClient
+{
+    public class FullPostModel
+    {
+        public int UserId { get; set; }
+
+        public int Id { get; set; }
+
+        public string Title { get; set; }
+
+        public string Body { get; set; }
+
+        public FullPostModel(int userId, int id, string title, string body)
+        {
+            UserId = userId;
+            Id = id;
+            Title = title;
+            Body = body;
+        }
+    }
+
+    public class CreatePostModel
```

Comment:
We usually keep all classed as a separate file. Class and interface that implements class could be in one file